integration_test/dashboard_test.dart

```diff
@@ -16,7 +16,7 @@ final GoRouter _router = GoRouter(
     GoRoute(
         path: '/',
         builder: (BuildContext context, GoRouterState state) {
-          return const Dashboard();
```

Comment:
🤔  I guess we should rename the test file as well if its now testing the wallet lightning component instead of the dashboard.